tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
 commands=
   py.test tests \
     --cov {env:PACKAGE} \
```

Comment:
The package name is always the same. Instead of passing it as an environment variable, I think it's better to hard-code it here, because then it allows the users to simply call `tox` directly.

I know we have `make test`, but often I want to call `tox -e py27`, or `tox -- tests/test_foo.py`, so it's easier if I don't have to pass any env.